tests/compile.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+
+// Recompile baseline ink files with the current version of inklecate avilable
+// $PATH.
```

Comment:
hu? Is that part of the previous comment?

Reply:
Er, yeah… ? Haha, yes it's confusing, but it might go away, see other comment.